packages/app-extensions/src/formField/typeEditable/decimal.js

```diff
@@ -0,0 +1,12 @@
+import _get from 'lodash/get'
+
+export default {
+  getOptions: ({modelField}) => ({
+    postPointDigits: _get(modelField, 'validation.decimalDigits.postPointDigits', null),
+    prePointDigits: _get(modelField, 'validation.decimalDigits.prePointDigits', null),
+    minValue: _get(modelField, 'validation.numberRange.fromIncluding', null),
+    maxValue: _get(modelField, 'validation.numberRange.toIncluding', null),
+    decimalScale: 28,
```

Comment:
where is this number coming from?

Reply:
+/-7.92282E+28 is max/min of type decimal.